zuul.d/jobs.yaml

```diff
@@ -9,65 +9,91 @@
       - github.com/openstack-k8s-operators/ci-framework
     vars:
       roles_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/edpm-ansible'].src_dir }}/roles/{{ TEST_RUN }}"
+      ANSIBLE_CALLBACKS_ENABLED: profile_tasks
+      ANSIBLE_VERBOSITY: 2
 
 # non-containerized role jobs
 - job:
     name: edpm-ansible-molecule-edpm_bootstrap
     parent: edpm-ansible-molecule-base
     vars:
       TEST_RUN: edpm_bootstrap
+      ANSIBLE_CALLBACKS_ENABLED: profile_tasks
+      ANSIBLE_VERBOSITY: 2
```

Comment:
Aren't the vars inherited from the parent job? Or am I reading this wrong.

https://zuul-ci.org/docs/zuul/latest/config/job.html